### PR TITLE
Remove validation

### PR DESCRIPTION
**Pull request by mennoo1996**
Descriptions are no longer mandatory to fill in

### CODE REVIEW

File: src/main/java/data/Camera.java [hunk 1]
```diff
@@ -163,4 +163,17 @@ public boolean equals(Object o) {
             return camera.ip == null;
         }
     }
+    
+    /**
+     * Construct string to display this camera in a modal.
+     * @return the correct string
+     */
+    public String constructModalString() {
```
Comment (bart-de-jonge) [inline src/main/java/data/Camera.java]: I see this method 3 times exactly the same, could you maybe extract it and make it a static method or something?

**Comment by bart-de-jonge** [comment]
One comment, please take a look

File: src/main/java/data/Camera.java (hunk 1 above)
Comment (mennoo1996) [inline src/main/java/data/Camera.java]: It would require making a class only for this method, which seems pretty overkill to me

**Comment by Markvanderuit** [comment]
Looks good to me, and I agree that making a separate class for that function would be overkill.
@bartdejonge1996 Any more comments, or shall I merge?

**Comment by bart-de-jonge** [comment]
Okay I will fix it myself then, duplicate code is no option

**Comment by bart-de-jonge** [comment]
Updated the code, extracted the method to projectcontroller as it is called the most there

**Comment by Markvanderuit** [comment]
Okay, so it was extracted to an existing class. I guess that works well. Nice work!

**Comment by mennoo1996** [comment]
Maybe you can just suggest putting it in an existing class next time instead of passive aggressively fixing it yourself. Could have done that as well

**Pull request merged**